broadinstitute/single_cell_portal_core
# greying and translucent unspecified label points (SCP-3782)

SCP-3782.  This grays and makes more translucent points with unspecified labels.  
![image](https://user-images.githubusercontent.com/2800795/157922612-6b7ae586-179d-4796-95d7-f1caf48e1823.png)

See the [design doc](https://docs.google.com/document/d/100mDxhnJgQpFjBDVxTHKRnsxdSpsY9stl1oycmlmkDg/edit#) for more details about why this implementation path was chosen.

TO TEST:
1. Load a study with a number of 'unspecified' points.  (e.g. cluster2 of the human_lymph synthetic study)
2. confirm the points appear as gray
3. increase the point size of the study so that the points overlap (for the human_lymph study, increasing the point size to ~20 does the trick
4. confirm the points under the unspecified points appear.
